.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
```

Comment:
It's better to switch to Ubuntu 16.04. trusty is 14.04
dist: xenial

Reply:
@xwu99  Thanks for your suggest , I will try it.

Reply:
@xwu99  I have switched to Ubuntu 16.04 , but it didn't work well .

Reply:
> @xwu99 I have switched to Ubuntu 16.04 , but it didn't work well .

The error "localhost: Error: JAVA_HOME is not set and could not be found."  We can easily fix this.

Reply:
@xwu99  Through the echo $JAVA_HOME command, we can get java env , but this error will be reported when starting hadoop.

Reply:
@xwu99  it's done , set JAVA_HOME in hadoop_env.sh.